src/structure.ts

```diff
@@ -445,8 +476,13 @@ interface StructureLab extends OwnedStructure<STRUCTURE_LAB> {
     mineralType: _ResourceConstantSansEnergy | null;
     /**
      * The total amount of minerals the lab can contain.
+     * @deprecated An alias for lab.store.getCapacity(lab.mineralType || yourMineral).
      */
     mineralCapacity: number;
+    /**
+     * A Store object that contains cargo of this structure.
+     */
+    store: GenericStore;
```

Comment:
Labs cannot have all type of resources, so this should be replaced by:
`store: Store<RESOURCE_ENERGY | MineralConstant | MineralCompoundConstant, false>;`
(the types I use here are on master now, so this will require merging from master)